kitchen/resources/demo.html

```diff
@@ -0,0 +1,245 @@
+<html>
```

Comment:
- Change the background to a solid background.
- Align text box with the search button
- Align classification text with the bottom of the box.

Did you write this from scratch?

Reply:
I used an example available online and used their html and css.

Reply:
Aligned text box with seach button.

Need to fix on the others.